external/fv3fit/fv3fit/reservoir/one_dim/domain_1d.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+from ..domain import slice_along_axis
+
+
+def _slice(arr: np.ndarray, inds: slice, axis: int = 0):
+    # https://stackoverflow.com/a/37729566
+    # For slicing ndarray along a dynamically specified axis
+    # same as np.take() but does not make a copy of the data
+    sl = [slice(None)] * arr.ndim
+    sl[axis] = inds
+    return arr[tuple(sl)]
```

Comment:
Can probably remove this implementation, since it is now imported from `..domain`.